lib/services/local/presence.go

```diff
@@ -610,18 +610,37 @@ func (s *PresenceService) CreateRemoteCluster(rc services.RemoteCluster) error {
 	return nil
 }
 
-// UpdateRemoteCluster updates remote cluster
+// UpdateRemoteCluster updates selected remote cluster fields: expiry and labels
+// other changed fields will be ignored by the method
 func (s *PresenceService) UpdateRemoteCluster(ctx context.Context, rc services.RemoteCluster) error {
-	value, err := json.Marshal(rc)
+	if err := rc.CheckAndSetDefaults(); err != nil {
+		return trace.Wrap(err)
+	}
+	existingItem, update, err := s.getRemoteCluster(rc.GetName())
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	item := backend.Item{
-		Key:     backend.Key(remoteClustersPrefix, rc.GetName()),
-		Value:   value,
-		Expires: rc.Expiry(),
+	update.SetExpiry(rc.Expiry())
+	update.SetLastHeartbeat(rc.GetLastHeartbeat())
+	meta := rc.GetMetadata()
+	meta.Labels = rc.GetMetadata().Labels
+	update.SetMetadata(meta)
```

Comment:
```suggestion
	update.SetMetadata(rc.GetMetadata())
```

Reply:
I don't want to update everything, just labels

Reply:
Ahh, in that case:
```suggestion
	meta := update.GetMetadata()
	meta.Labels = rc.GetMetadata().Labels
	update.SetMetadata(meta)
```